packages/realm/binding/android/CMakeLists.txt

```diff
@@ -1,128 +1,74 @@
 cmake_minimum_required(VERSION 3.18.1)
+project(RealmJS)
 
-set(REACT_NATIVE_ROOT_DIR "${PACKAGE_ROOT_DIR}/node_modules/react-native")
-set(REACT_NATIVE_AAR_DIR "${CMAKE_BINARY_DIR}/../react-native-aar") # TODO: Could we use CMAKE_BINARY_DIR instead?
-set(REACT_NATIVE_VERSION "0.71.0")
-
-set(JSI_HEADER_DIR "${REACT_NATIVE_ROOT_DIR}/ReactCommon/jsi")
+# Use ccache if available
+find_program(CCACHE_PROGRAM ccache)
+if(CCACHE_PROGRAM)
+    set_property(GLOBAL PROPERTY RULE_LAUNCH_COMPILE "${CCACHE_PROGRAM}")
+endif()
 
-add_library(realm-js-android SHARED
-    # $<TARGET_OBJECTS:realm-js-shared>
-    hack.cpp
-    platform.cpp
-    jni_utils.cpp
-    io_realm_react_RealmReactModule.cpp
-    ${REACT_NATIVE_ROOT_DIR}/ReactAndroid/src/main/jni/react/turbomodule/ReactCommon/CallInvokerHolder.cpp
+add_library(realm-js-android-binding SHARED
+    ../jsi/jsi_init.cpp
+    src/main/cpp/hack.cpp
```

Comment:
I think we can remove this hack, since it's only targeting `armeabi`/`ARMv5` 32-bit Android which was removed in NDK r17